projects.json

```diff
@@ -2721,6 +2721,61 @@
       }
     ]
   },
+  {
+  "repository": "Git",
+  "url": "https://github.com/masters3d/xswift.git",
+  "path": "xswift",
+  "branch": "do-not-delete-pre-4.2-compatibility",
+  "maintainer": "cheyo@masters3d.com",
+  "compatibility": [
+    {
+      "version": "4.0",
+      "commit": "7614aae4a87d92e2343473c619a805ace98a474e"
+    },
+    {
+      "version": "4.1",
+      "commit": "7614aae4a87d92e2343473c619a805ace98a474e"
+    }
+  ],
+  "platforms": [
+    "Darwin"
+  ],
+  "actions": [
+    {
+      "action": "BuildXcodeProjectTarget",
+      "project": "xswift.xcodeproj",
+      "target": "xswift-Package",
+      "destination": "generic/platform=macOS",
+      "configuration": "Release"
+    }
+  ]
+  },
+  {
+  "repository": "Git",
+  "url": "https://github.com/exercism/swift.git",
+  "path": "xswift",
```

Comment:
Having two projects with the same 'path' is not supported. This should be renamed. I renamed this to 'exercism' and ran it locally (needs Xcode 10 Beta 3), and the 4.2 configuration passed.

I've updated project_procommit_check in #222. As soon as that merges, you should be able to reproduce this as follows.

```
./project_precommit_check exercism --earliest-compatible-swift-version 4.2
** CHECK **
--- Validating exercism Swift version 4.2 compatibility ---
--- Project configured to be compatible with Swift 4.2 ---
--- Checking exercism platform compatibility with Darwin ---
--- Platform compatibility check succeeded ---
--- Locating swiftc executable ---
$ xcrun -f swiftc
--- Checking installed Swift version ---
$ /Applications/Xcodes/Xcode_10_Beta_3.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/swiftc --version
--- Version check succeeded ---
--- Executing build actions ---
$ ./runner.py --swift-branch swift-4.2-branch --swiftc /Applications/Xcodes/Xcode_10_Beta_3.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/swiftc --projects ./projects.json --include-repos 'path == "exercism"' --include-versions 'version == "4.2"' --include-actions 'action.startswith("Build")'
PASS: exercism, 4.2, ce9509, Swift Package
========================================
Action Summary:
     Passed: 1
     Failed: 0
    XFailed: 0
    UPassed: 0
      Total: 1
========================================
Repository Summary:
      Total: 1
========================================
Result: PASS
========================================
--- exercism checked successfully against Swift 4.2 ---
```